Source/WebKit/Shared/Cocoa/WKObject.h

```diff
@@ -59,8 +60,7 @@ template<typename WrappedObjectClass> struct WrapperTraits;
 
 template<typename DestinationClass, typename SourceClass> inline DestinationClass *checkedObjCCast(SourceClass *source)
 {
-    ASSERT([source isKindOfClass:[DestinationClass class]]);
-    return (DestinationClass *)source;
+    return checked_objc_cast<DestinationClass>(source);
```

Comment:
Hm... this introduces `isKindOfClass` obj-c message and causes trivial function detector to bail out.
Will revert this portion for now.